components/camel-as2/camel-as2-api/src/test/java/org/apache/camel/component/as2/api/AS2MessageTest.java

```diff
@@ -115,6 +114,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(AS2MessageTest.class);
 
     private static final String METHOD = "POST";
+    private static final String RECIPIENT_DELIVERY_ADDRESS = "http://localhost:8080/handle-receipts";
     private static final String TARGET_HOST = "localhost";
     private static final int TARGET_PORT = 8080;
```

Comment:
It would be better to use AvailablePortFinder.getNextAvailable() instead of hardcoding port number to prevent port conflicts on different envs.